Source/GSTitleView.m

```diff
@@ -238,9 +239,24 @@ - (void) mouseDown: (NSEvent*)theEvent
   NSDate   *theDistantFuture = [NSDate distantFuture];
   NSPoint  startWindowOrigin;
   NSPoint  endWindowOrigin;
+  CGFloat  leftLimit;
+  CGFloat  topLimit;
+  CGFloat  rightLimit;
+  CGFloat  bottomLimit;
 
   NSDebugLLog (@"NSMenu", @"Mouse down in title!");
 
+  // Define move constrains for menu
+  if (_ownedByMenu)
+    {
+      NSRect screenFrame = [[NSScreen mainScreen] frame];
```

Comment:
As I already wrote in a private mail, we should be using [_window screen] here. There is even a method on NSWindow to handle this code constrainFrameRect: toScreen:, why not use that?

Reply:
> As I already wrote in a private mail, we should be using [_window screen] here. 

Fixed.

> There is even a method on NSWindow to handle this code constrainFrameRect: toScreen:, why not use that?

Apple documentation states: "If the window is resizable and the window’s height is greater than the screen height, the rectangle’s height is adjusted to fit within the screen as well. The rectangle’s width and horizontal location are unaffected." This is not I want to achieve. I want constrains at left and right also. Plus, I want user to be able move menu off the screen while menu title should remain visible when moving menu to the bottom.